docs/UserGuide.md

```diff
@@ -26,33 +30,35 @@ Store the information of key contacts in InterNUS. During the internship-hunting
 Track the internships you're interested in and all the relevant information -- the company, role, contact person and interview date. Awaiting a reply, or have you been accepted? With InterNUS, you can monitor and update the statuses of your internship applications effortlessly.
 
 --------------------------------------------------------------------------------------------------------------------
-# 2.How to use this User Guide?
+# 2. How to use this User Guide?
 
-## 2.1 Icons and symbols
+1. For new user, proceed to our [Quick start](#3-quick-start)  
 
-## 2.2 Graphical User Interface (GUI) of InterNUS
+1. For returning user, skip to our [Command summary](#command-summary) for a summary of our feature
 
-## 2.3 Command format
 
---------------------------------------------------------------------------------------------------------------------
-# 3. Quick start
-
-1. Ensure you have Java `11` or above installed in your Computer.
-
-1. Download the latest `InterNUS.jar` from [here](https://github.com/AY2223S1-CS2103T-F11-1/tp/releases).
-
-1. Copy the file to the folder you want to use as the _home folder_ for your app.
-
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+## 2.1 Icons and symbols
+| Symbol               | Meaning                                                                                      |
+|----------------------|----------------------------------------------------------------------------------------------|
+| :information_source: | Additional information that may be useful to know when using InterNUS                        |
+| :exclamation:        | Important information or warnings that you should take note of when using InterNUS           | 
+| `command`            | The highlighted words indicate commands that can be typed into the Command Input of InterNUS |
```

Comment:
I think we shouldn't specify this since we use the `highlighted format` in many other cases.
```suggestion
```